culture/remote/README.md

```diff
@@ -0,0 +1,43 @@
+# Working with our Remote Teammates
+
+At Sparkbox, we have the opportunity to work with many different people in many different locations. Whether its' with our co-workers or our clients, sometimes, we all have to work remote.
+
+## Tips for Non-remote folks When Working With Remote Colleagues
+
+Let's be more inclusive of our remote coworkers and clients. Here are some ways how.
+
+- When a conversation becomes more than a quick back-and-forth on [Slack][], spin up a [Zoom][] and encourage all parties to join.
+- Join meetings 2-5 minutes early, to encourage friendly, human banter. Avoid jumping directly into tasking whenever possible.
+- Use a headset/headphone in public spaces. Avoid noisy or high-traffic areas, even if they don't seem loud to you.
+- Let people know if you're not using a headset, or if they are "on blast" in public spaces.
+- Activities with physical paper, whiteboards, or other items should be avoided. Use online tools such as [draw.io][] instead.
+- When one person is remote on a meeting, everyone should be remote. Find individual spaces, such as your desk, so that everyone uses headphones and is on the call separately. This equalizes the speaking floor so that everyone involved has the same opportunity to share and listen. No person has to compete with a large conference room where the video screen is quickly forgotten.
+- If you _are_ in a situation with a big group in one room and a few remote people on a call, designate a "point person" in the large group space as an advocate for your remote friends.
```

Comment:
What should this remote advocate do to advocate?

Reply:
I'm a big fan of a rotating advocate who is responsible for digital notes, taking and posting frequent pics if there is something physical to be seen, and watching to see when someone wants to say something remotely. That seems easy to flow in and has side benefits. I've been trialing this with remote Discoveries and feel it has worked really well.

Reply:
Seeing the concerns on the note above, I'd be happy to talk with anyone about how we've been doing advocates in some of my meetings. We've had really, really positive feedback from participants in the last one that Austin was the advocate for (so maybe you want to talk to her). It requires discipline, but this does work well when practiced.

Reply:
I'd love a demo at all-hands about how to be a remote advocate